content/kotlin/concepts/queue/queue.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'Queue'
+Description: 'A Queue is a collection interface that offers additional operations for accessing or manipulating items at the head of the queue.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Collections'
+  - 'Queues'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+A **`Queue`** is an interface in Kotlin that represents a data structure that stores elements in a linear order, and follows the First In, First Out (FIFO) principle. This means that the first element added to the queue will be the first one to be removed.
+
+### Syntax
+
+The syntax for creating a `Queue` in Kotlin using the `LinkedList` class is as follows:
+
+```pseudo
+val queue = LinkedList<Type>()
+```
+
+Where `Type` is the type of elements that the `Queue` will hold, for example, to store integers, use `Int` as the type :
```

Comment:
```suggestion
`Type` is the type of elements that the `Queue` will hold. For example, to store integers, use `Int` as the type:
```

Maybe instead of creating a new pseudo block, we can list the possible types (including `Int`) and link out to the data types entry. In addition, I think the queue interface pseudo block can be appended to the first pseudo block (so it's just one block). This can be explained following the syntax/pseudo block.